src/Module.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace froala\froalaeditor;
```

Comment:
Let's user Froala\Editor

Reply:
Not sure if I understand this change, do you want something like this? 

![image](https://user-images.githubusercontent.com/3048593/30312677-28e64a5e-97b8-11e7-9a25-6541f5060e1d.png)

Reply:
Yes, it would be better to use `Froala\Editor` instead of `froala\froalaeditor`.